tracee-ebpf/main.go

```diff
@@ -114,12 +116,30 @@ func main() {
 				fmt.Fprintf(os.Stderr, "warning: could not check enabled kconfig features, trying to continue anyway\n")
 			}
 
-			// try to discover distro by /etc/os-release, fallback to kernel version
+			// OS release information
+
+			osReleaseFilePath, err := checkEnvPath("TRACEE_OSRELEASE_FILE") // useful if users wants to mount host os-release in a container
+			if err != nil {
+				return err
+			} else if osReleaseFilePath == "" {
+				osReleaseFilePath = "/etc/os-release"
+			}
```

Comment:
maybe pull this into NewOSInfo so that it's interface is simpler - in case of no args user has to call NewOSInfo("")